pkg/connector/schedule.go

```diff
@@ -219,6 +220,8 @@ func (s *scheduleResourceType) Grants(ctx context.Context, resource *v2.Resource
 					},
 				),
 			)
+		case escalationParticipantType:
+			resourceType = resourceTypeEscalation.Id
```

Comment:
I think until we add proper escalation support, we might just want to `continue` here to avoid the error

Reply:
Would this be a valuable place to log info?